core/app/components/PubRow.tsx

```diff
@@ -96,24 +92,14 @@ const PubRow: React.FC<Props> = async (props: Props) => {
 							{pub.pubType.name}
 						</div>
 						<div className="flex flex-row gap-x-2">
-							{pubStage && (
-								<Suspense>
-									<IntegrationActions
-										pubId={pub.id}
-										token={props.token}
-										stageId={pubStage.id}
-										type="pub"
-									/>
-								</Suspense>
-							)}
 							<div>{props.actions}</div>
 							<PubDropDown pubId={pub.id} searchParams={props.searchParams} />
 						</div>
 					</div>
 				</RowHeader>
 				<RowContent className="flex items-start justify-between">
 					<h3 className="text-md font-medium">
-						<Link href={`pubs/${pub.id}`} className="hover:underline">
```

Comment:
if you're here already, any change of changing this to `/pubs/${communitySlug}/${pub.id}`? Always found this intermediary step kinda weird since we have no real reason to link to other communities' pubs yet.